netebpfext/net_ebpf_ext.h

```diff
@@ -81,16 +81,22 @@ typedef struct _net_ebpf_extension_wfp_filter_context
     const struct _net_ebpf_extension_hook_client* client_context; ///< Pointer to hook NPI client.
     uint64_t* filter_ids;                                         ///< Array of WFP filter Ids.
     uint32_t filter_ids_count;                                    ///< Number of WFP filter Ids.
+    bool client_detached;                                         ///< True if client has detached.
```

Comment:
imho, should we increase the number of bool-s, the code using it would not change, in case this is later converted to a bitfield union (i.e., same would apply to `_net_ebpf_ext_hook_client_rundown`).

Reply:
My preference would be for a bitfield instead of boolean as well.  IMO a bit serves the purpose well enough, especially when there's a distinct possibility of more to be added.